### PR TITLE
DOC-1325: RS CLI Reorg: rladmin node

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/jira-doc-1325/rs/references/cli-utilities/rladmin/node/

Still working on the individual command pages.

### CODE REVIEW

File: content/rs/references/cli-utilities/rladmin/node/addr.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| Parameter | Type/Value                     | Description                                                                                   |
+|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|
+| node      | node ID                        | Sets internal IP address of the specified node                                                |
+| ip_addr   | IP address                     | Sets specified IP address as the internal IP of the node                                      |
+
+### Returns
+
+Returns `Updated successfully` if the IP address was set.
+
+Use [`rladmin status nodes`]({{<relref "/rs/references/cli-utilities/rladmin/status#status-nodes">}}) to verify the internal IP address was changed.
```
Comment (nermiller) [inline content/rs/references/cli-utilities/rladmin/node/addr.md:31]: Consider starting with action first, i.e. "Verify that the internal IP address has changed, ..."

File: content/rs/references/cli-utilities/rladmin/node/addr.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+description: Sets the internal IP address of a node.
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin node addr set` sets the internal IP address of a node. The internal IP address can only be set with `rladmin node addr` when the node is down.
```
Comment (nermiller) [inline content/rs/references/cli-utilities/rladmin/node/addr.md:13]:
```suggestion
`rladmin node addr set` sets the internal IP address of a node. You can set the internal IP address using `rladmin node addr` only when the node is down.
```

**Review by nermiller — APPROVED** [review]
LGTM with minor suggestions

File: content/rs/references/cli-utilities/rladmin/node/addr.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+rladmin node <id>
+        addr set <ip_addr>
+```
+
+### Parameters
+
+| Parameter | Type/Value                     | Description                                                                                   |
+|-----------|--------------------------------|-----------------------------------------------------------------------------------------------|
+| node      | integer                        | Sets internal IP address of the specified node                                                |
+| ip_addr   | IP address                     | Sets specified IP address as the internal IP of the node                                      |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/node/addr.md:25]:
```suggestion
| node      | integer                        | Sets the internal IP address of the specified node                                                |
| ip_addr   | IP address                     | Sets the node's internal IP address to the specified IP address                                      |
```

File: content/rs/references/cli-utilities/rladmin/node/enslave.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                [retry_timeout_seconds]
+```
+
+### Parameters
+
+| Parameter             | Type/Value                     | Description                                                                               |
+|-----------------------|--------------------------------|-------------------------------------------------------------------------------------------|
+| node                  | integer                        | Changes all of the node's endpoints and shards to replicas             |
+| demote_node           |                                | If the node is a primary node, changes the node to replica                     |
+| retry_timeout_seconds | integer                        | Retries any failures until the specified number of seconds has passed.                    |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/node/enslave.md:31]:
```suggestion
| retry_timeout_seconds | integer                        | Retries on failure until the specified number of seconds has passed.                    |
```

File: content/rs/references/cli-utilities/rladmin/node/enslave.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+$ rladmin node <id> enslave endpoints_only
+                [retry_timeout_seconds]
+```
+
+### Parameters
+
+| Parameter             | Type/Value                     | Description                                                                               |
+|-----------------------|--------------------------------|-------------------------------------------------------------------------------------------|
+| node                  | integer                        | Changes all of the node's endpoints to replicas                        |
+| retry_timeout_seconds | integer                        | Retries any failures until the specified number of seconds has passed.                    |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/node/enslave.md:87]:
```suggestion
| retry_timeout_seconds | integer                        | Retries on failure until the specified number of seconds has passed.                    |
```

File: content/rs/references/cli-utilities/rladmin/node/enslave.md [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+$ rladmin node <id> enslave shards_only
+                [retry_timeout_seconds]
+```
+
+### Parameters
+
+| Parameter             | Type/Value                     | Description                                                                               |
+|-----------------------|--------------------------------|-------------------------------------------------------------------------------------------|
+| node                  | integer                        | Changes all of the node's shards to replicas                          |
+| retry_timeout_seconds | integer                        | Retries any failures until the specified number of seconds has passed.                    |
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/node/enslave.md:127]:
```suggestion
| retry_timeout_seconds | integer                        | Retries on failure until the specified number of seconds has passed.                    |
```

File: content/rs/references/cli-utilities/rladmin/node/enslave.md [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+`rladmin node enslave` changes the resources of a node to replicas.
+
+## `node enslave`
+
+Changes all of the node's endpoints and shards to replicas.
+
+``` sh
+$ rladmin node <id> enslave
+                [demote_node]
+                [retry_timeout_seconds]
```
Comment (rrelledge) [inline content/rs/references/cli-utilities/rladmin/node/enslave.md:22]: `retry_timeout_seconds` accepts an integer, right? If so, the other two commands need to be updated too.
```suggestion
                [retry_timeout_seconds] <seconds>
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
These are looking good! I left some suggestions for changes.

**Review by rrelledge — APPROVED** [review]
Looks good to me.

**Pull request merged**